PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cff.py

```diff
@@ -18,3 +18,25 @@
     patJets
     )
 makePatJets = cms.Sequence(makePatJetsTask)
+
+from RecoBTag.ImpactParameter.pfImpactParameterTagInfos_cfi import * #pfImpactParameterTagInfos
+from RecoBTag.SecondaryVertex.pfSecondaryVertexTagInfos_cfi import * #pfSecondaryVertexTagInfos
+from RecoBTag.SecondaryVertex.pfInclusiveSecondaryVertexFinderTagInfos_cfi import * #pfInclusiveSecondaryVertexFinderTagInfos
+from RecoBTag.Combined.deepFlavour_cff import * #pfDeepFlavourTask
+from Configuration.Eras.Modifier_run2_miniAOD_80XLegacy_cff import run2_miniAOD_80XLegacy
+
+#make a copy to avoid labels and substitution problems
+_makePatJetsCopy = makePatJets.copy()
+
+run2_miniAOD_80XLegacy.toReplaceWith(
+    makePatJets, 
+    cms.Sequence(
```

Comment:
I still think that it's the task that should be modified, not the Sequence.
The sequence is a derived object. So, it probably should not even be modified explicitly.
If I'm not mistaken, if a sequence is present, the task should still be defined.

Was this last version checked to work?